index.js

```diff
@@ -443,7 +443,7 @@ const fileType = input => {
 			check([0xFF, 0xF0], {offset: start, mask: [0xFF, 0xFC]}) // MPEG 4 layer 0 using ADTS
 		) {
 			return {
-				ext: 'mp4',
+				ext: 'aac',
 				mime: 'audio/mpeg'
```

Comment:
Media type should probably be `audio/aac`, it's standardized.

Reply:
> Media type should probably be audio/aac, it's standardized.

I think you are right: ([ref](https://www.w3.org/2013/12/byte-stream-format-registry/mpeg-audio-byte-stream-format.html#h2_mime-types))

> "audio/aac" for sequences of ADTS frames, as specified in ISO/IEC 14496-3:2009